test/elm/comparison/test.coffee

```diff
@@ -93,6 +93,12 @@ describe 'Equal', ->
   it 'should be null for 5 Cel = 40 m', ->
     should(@aLtB_Quantity_incompatible.exec(@ctx)).be.null()
 
+  it 'should be true for 10mg:2dL = 15mg:3dL', ->
+    @eqRatios.exec(@ctx).should.be.true()
+  
+  it 'should be true for 10mg:2dL = 15mg:4dL', ->
```

Comment:
typo: it should be _false_

Reply:
<facepalm> I shouldn't have let that go through